bin/windows-toolset.ps1

```diff
@@ -163,11 +151,29 @@ if ($buildSource) {
 cd $toolsDir
 
 # Download openssl
-Write-Host "Downloading http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma"
-(New-Object net.webclient).DownloadFile("http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma", "$toolsDir\${opensslPkg}.tar.lzma")
+Write-Host "Downloading https://s3.amazonaws.com/kylo-pl-bucket/${opensslPkg}.tar.lzma"
```

Comment:
I just put ruby up on buildsources, so this is very do-able if that's what we want. @stahnma do you want us to start putting curl, boost, and yaml-cpp (which are build with this script) there as well? I think those are the ones that are needed by AppVeyor